src/main/java/frc/robot/subsystems/arm/ArmBase.java

```diff
@@ -45,20 +39,9 @@ public ArmBase() {
         baseMotor.setSelectedSensorPosition(angleToTick(90));
         feedforward = new ArmFeedforward(kS_BASE, kG_BASE, kV_BASE, kA_BASE);
     }
-    
-    /**private void configCancoder() {
-        CANCoderConfiguration config = new CANCoderConfiguration();
-        config.absoluteSensorRange = AbsoluteSensorRange.Unsigned_0_to_360;
-        config.magnetOffsetDegrees = 0;
-        config.sensorDirection = false;
-        config.initializationStrategy = SensorInitializationStrategy.BootToAbsolutePosition;
-        canCoder.configAllSettings(config);
-        canCoder.setPosition(
-                canCoder.getAbsolutePosition() - BASE_CC_OFFSET);
-    }*/
```

Comment:
You shouldn’t delete this method, as it will likely prove useful once the CANCoder is added again. For now, just uncomment it. Since you aren’t referencing it anywhere else, it will never be executed, so you don’t have to worry about the device not existing. (This is called “dead code”)